### ISSUE TITLE
Downloading latest version of the addon using wget tool

### ISSUE THREAD

**Issue by BoQsc** [mozilla/addons#805]
### Describe the problem and steps to reproduce it:
There is no guide or information about how to retrieve latest addon using wget tool.

Just a quick example:

- open in a Firefox browser: https://addons.mozilla.org/en-US/firefox/addon/gnome-shell-integration/
- Hover over button named **+ Add to Firefox**
- See that the link points to specific version of the addon.

### What happened?
I opened a random addon page, and there is link to **specific** .xpi version of the addon instead of latest version.
`https://addons.mozilla.org/firefox/downloads/file/898030/gnome_shell_integration-10.1-an+fx-linuz.xpi`

### What did you expect to happen?
Getting link to the latest version, instead of specific version.
`https://addons.mozilla.org/firefox/downloads/file/gnome_shell_integration-an+fx-linuz-latest.xpi`

**Comment by diox** [comment]
Hi,

This is a duplicate of https://github.com/mozilla/addons-frontend/issues/4083, which itself is blocked by https://github.com/mozilla/addons-server/issues/7187.
